string-manipulation-1/src/is-vowel.js

```diff
@@ -0,0 +1,18 @@
+/* exported isVowel */
+/*
+goal - return a boolean true if letter is vowel and false if letter is constanant
+check letter in input
+if letter is Vowel, return true
+else, return false
+
+*/
+
+function isVowel(char) {
+  if (char === 'A' || char === 'E' || char === 'I' || char === 'O' || char === 'U') {
+    return true;
+  } else if (char === 'a' || char === 'e' || char === 'i' || char === 'o' || char === 'u') {
```

Comment:
You should just put all these conditions in one `if`, there's no need to separate it into separate blocks, since the work is the same for both.